helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/PerInstanceAccessor.java

```diff
@@ -435,6 +435,11 @@ public Response updateInstance(@PathParam("clusterId") String clusterId,
       case setInstanceOperation:
          admin.setInstanceOperation(clusterId, instanceName, state);
          break;
+        case completeSwapIfPossible:
```

Comment:
This apparently is the correct indentation according to helix style configuration; however, the others are incorrect. I didn't change the others because want to keep the diff only on my changes.